<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Horarios</class>
 <widget class="QMainWindow" name="Horarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Horários</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>951</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Editar">
     <attribute name="title">
      <string>Editar</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>250</y>
        <width>191</width>
        <height>301</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>9</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <row>
       <property name="text">
        <string>Entrada1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Saida1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Entrada2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Saida2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Entrada3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Saida3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Hora Extra</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Ausência</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Comentário</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Valores</string>
       </property>
      </column>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>341</width>
        <height>221</height>
       </rect>
      </property>
      <property name="verticalHeaderFormat">
       <enum>QCalendarWidget::NoVerticalHeader</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>310</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Almoço no CNPEM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_save">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>431</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>150</y>
        <width>241</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Compensação de pontes e feriados</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>109</y>
         <width>31</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Fim:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>70</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Início:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="de_begin">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>110</width>
         <height>27</height>
        </rect>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2018</year>
         <month>1</month>
         <day>2</day>
        </date>
       </property>
      </widget>
      <widget class="QDateEdit" name="de_end">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>110</width>
         <height>27</height>
        </rect>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2018</year>
         <month>10</month>
         <day>11</day>
        </date>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_configure">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>140</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Configurar</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pb_calculate">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>391</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_update">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>350</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Hoje</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_clear">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>470</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Visualizar</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>921</width>
        <height>361</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Entrada1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Saida1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Entrada2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Saida2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Entrada3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Saida3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Hora Extra</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ausência</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Comentário</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Almoço</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Compensação</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>25</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>De:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>24</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Até:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="de_from">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>110</width>
        <height>27</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>dd/MM/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2018</year>
        <month>3</month>
        <day>15</day>
       </date>
      </property>
     </widget>
     <widget class="QDateEdit" name="de_to">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>110</width>
        <height>27</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>dd/MM/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2018</year>
        <month>3</month>
        <day>15</day>
       </date>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>474</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Horas Positivas:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>493</x>
        <y>519</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Horas Negativas:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_positive">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>470</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_negative">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>514</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>123</x>
        <y>474</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Hora Extra Total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>133</x>
        <y>520</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Ausência Total:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_he">
      <property name="geometry">
       <rect>
        <x>223</x>
        <y>470</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_absent">
      <property name="geometry">
       <rect>
        <x>223</x>
        <y>514</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_load">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Carregar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
